src/uk/gov/hmcts/contino/GradleBuilder.groovy

```diff
@@ -68,12 +68,12 @@ class GradleBuilder extends AbstractBuilder {
     }
   }
 
-  def crossBrowserTest() {
+  def crossBrowserTest(String browser = "") {
```

Comment:
this doesn't seem to match your description, you've put an array in the example:
```
enableCrossBrowserTest(['chrome', 'firefox', 'safari', 'microsoft'])
```

but this only accepts a string and you're only testing a string

Reply:
Good question.
So this is an extension of [existing functionality](https://github.com/hmcts/cnp-jenkins-library/blob/master/vars/sectionNightlyTests.groovy#L53-L70):
```
      browsers.each { browser ->
        crossBrowserStages.put(browser.capitalize(), {
          warnError('Failure in crossBrowserTest') {
            pcr.callAround('crossBrowserTest') {
              timeoutWithMsg(time: config.crossBrowserTestTimeout, unit: 'MINUTES', action: 'Cross browser test') {
                builder.crossBrowserTest(browser)
              }
            }
          }
        })
```
which takes that defined array of strings and makes individual calls to the builder for each string. 
However, until this change, there was only a compatible method in the Yarn builder.

I'll add this detail to the description to make it clearer

Reply:
I would recommend making this implementation consistent with the YarnBuilder one, duplicating the method rather than using the groovy default parameter, it would end up with less code / clearer what's going on imo.

If you feel strongly it's not clearer then I would ask that you adjust YarnBuilder to be consistent, thanks!

Reply:
happy to follow the Yarn format if it improves readability, even if it increases the code slightly. I'll do that now

Reply:
done